<launch>


<!-- modification a faire -->

  <node name="kalman_node" pkg="seabot_fusion" type="kalman_node" output="screen" ns="fusion" respawn="true" respawn_delay="5">
				<param name="frequency" value="5" />
				<param name="limit_min_depth" value="0.3" />

				<param name="estimated_first_error_equilibrium_tick" value="250" />

				<param name="gamma_alpha_velocity" value="1e-4" />
				<param name="gamma_alpha_depth" value="1e-5" />
				<param name="gamma_alpha_offset" value="1e-8" />
				<param name="gamma_alpha_chi" value="1e-8" />
  </node>

</launch>
